### ISSUE TITLE
Doc Suggestion: Move Dark Theme from Typography to Colors, Classes, or Variables

### ISSUE THREAD

**Issue by jgentes** [palantir/blueprint#4750]
Just a quick suggestion - I had a hard time (until I searched, of course) finding the specifics for how to use the dark theme.

It seemed odd to me that it wasn't under Colors or Classes or Variables, but was located under Typography. Since it is a theme-wide setting, would it be better to move this elsewhere?

https://blueprintjs.com/docs/versions/4/#core/typography.dark-theme

Great job, btw!

**Comment by adidahiya** [comment]
Good point, I'll look into moving the section to make it more discoverable.

**Comment by adidahiya** [comment]
duplicate of #4750
